app/controllers/JobController.scala

```diff
@@ -20,6 +20,10 @@ class JobController @Inject()(
   connectionDAO: ConnectionDAO
 )(implicit ec: ExecutionContext) extends AbstractController(cc) with I18nSupport {
 
+  def list = silhouette.SecuredAction.async { implicit request =>
+    jobDAO.findAll(request.identity).map(jobs => Ok(Json.toJson(jobs)))
```

Comment:
Do you have any opinions on the response shape(s)? Should we be better at normalizing for all endpoints (nest under "result" key or something)?

Reply:
that sounds fine. should result always be an array? or can be object too?

Reply:
For this endpoint would always be array, but in general could be either. Overall payload would always be object.